client/app/dedicatedCloud/datacenter/drp/configuration/steps/ovh/secondPcc/dedicatedCloud-datacenter-drp-ovh-secondPccStep.controller.js

```diff
@@ -123,13 +130,62 @@ export default class {
     return this.$state.go('app.dedicatedClouds.datacenter.drp.ovh.mainPccStep', { drpInformations: this.drpInformations });
   }
 
-  goToNextStep() {
-    return this.$state.go('app.dedicatedClouds.datacenter.drp.ovh.confirmationStep', { drpInformations: this.drpInformations });
-  }
-
   isStepValid() {
     return this.selectedSecondaryPcc
       && this.drpInformations.secondaryDatacenter
       && this.selectedSecondaryIpAddress;
   }
+
+  validateConfiguration() {
+    this.isLoading = true;
+
+    return this.dedicatedCloudDrp.enableDrp(
+      this.drpInformations,
+      this.drpInformations.primaryPcc.generation !== this.PCC_NEW_GENERATION,
+    )
+      .then((enableDrp) => {
+        if (enableDrp.url !== undefined) {
+          this.storeZertoOptionOrderInUserPref(this.drpInformations, enableDrp);
+          if (!enableDrp.hasAutoPay) {
+            this.$window.open(enableDrp.url, '_blank');
+          }
+        }
+
+        return this.$state.go('app.dedicatedClouds', {}, { reload: true }).then(() => {
+          // $timeout necessary to display alerter message
+          this.$timeout(() => {
+            this.Alerter.set(
+              'alert-info',
+              `
+              ${this.$translate.instant('dedicatedCloud_datacenter_drp_confirm_creation_pending')} ${this.$translate.instant('dedicatedCloud_datacenter_drp_confirm_creation_pending_task')}
+            `,
+              null,
+              'dedicatedCloud_alert',
+            );
+          }, 0);
+        });
+      })
+      .catch((error) => {
+        this.Alerter.error(`${this.$translate.instant('dedicatedCloud_datacenter_drp_confirm_create_error')} ${_.get(error, 'data.message', error.message)}`, 'dedicatedCloudDatacenterDrpAlert');
+      })
+      .finally(() => {
+        this.loading = false;
+      });
+  }
+
+  storeZertoOptionOrderInUserPref(drpInformations, enableDrp) {
+    const drpInformationsToStore = {
+      drpInformations,
+      zertoOptionOrderId: enableDrp.orderId,
+      zertoOption: drpInformations.drpType === this.DEDICATEDCLOUD_DATACENTER_DRP_OPTIONS.ovh
+        ? this.DEDICATEDCLOUD_DATACENTER_DRP_ORDER_OPTIONS.zertoOption.ovh
+        : this.DEDICATEDCLOUD_DATACENTER_DRP_ORDER_OPTIONS.zertoOption.onPremise,
+    };
+
+    const { splitter } = this.DEDICATEDCLOUD_DATACENTER_ZERTO;
+    const [, ...[formattedServiceName]] = drpInformations.primaryPcc.serviceName.split(splitter);
+    const preferenceKey = `${this.DEDICATEDCLOUD_DATACENTER_ZERTO.title}_${formattedServiceName.replace(/-/g, '')}`;
```

Comment:
Looks similar to https://github.com/ovh-ux/ovh-manager-dedicated/pull/1237/files#diff-ec60f6e4a52486ece54e5247961787e3R79